code/modules/vehicles/mecha/equipment/weapons/mecha_ammo.dm

```diff
@@ -61,26 +61,14 @@
 	rounds = 300
 	ammo_type = "lmg"
 
-/obj/item/mecha_ammo/missiles_br
-	name = "breaching missiles"
-	desc = "A box of large missiles, ready for loading into a BRM-6 exosuit missile rack."
-	icon_state = "missile_br"
-	rounds = 6
-	round_term = "missile"
-	direct_load = TRUE
-	load_audio = 'sound/weapons/gun/general/mag_bullet_insert.ogg'
-	ammo_type = "missiles_br"
-
-/obj/item/mecha_ammo/missiles_he
-	name = "anti-armor missiles"
-	desc = "A box of large missiles, ready for loading into an SRM-8 exosuit missile rack."
-	icon_state = "missile_he"
```

Comment:
huh?

Reply:
the _br missiles were breaching missiles for the BRM-6

the _he missiles were the old missiles for the SRM-8
the SRM-8 got the kinetic missiles now